src/utilities/inputProcessors.spec.ts

```diff
@@ -0,0 +1,145 @@
+import * as actionsCore from '@actions/core';
+
+import {
+  processInputCommitAssets,
+  processInputDryRun,
+  processInputNodeModule,
+  processInputReleaseAssets,
+  processInputReleaseBranches,
+} from './inputProcessors';
+
+const getInputSpy = jest.spyOn(actionsCore, 'getInput').mockImplementation();
+
+describe('processInputNodeModule', (): void => {
+  it("returns true when the value of the npm-package input is set to 'true'", (): void => {
+    expect.assertions(1);
+
+    getInputSpy.mockReturnValue('true');
+
+    const result = processInputNodeModule();
+
+    expect(result).toStrictEqual(true);
+  });
+});
+
+describe('processInputDryRun', (): void => {
+  it("returns true when the value of the dry-run input is set to 'true'", (): void => {
+    expect.assertions(1);
+
+    getInputSpy.mockReturnValue('true');
+
+    const result = processInputDryRun();
+
+    expect(result).toStrictEqual(true);
+  });
+});
+
+describe('processInputReleaseBranches', (): void => {
+  it('throws an error if the input parameter value is set to an invalid JSON string', (): void => {
+    expect.assertions(1);
+
+    getInputSpy.mockReturnValue('test');
+
+    expect(processInputReleaseBranches).toThrow(
+      'Invalid JSON string for input parameter release-branches.',
+    );
+  });
+
+  it.each([
+    {
+      value: '[{}]',
+    },
+    {
+      value: '[{"name": 1}]',
+    },
+    {
+      value: '[{"name": "test", "channel": true}]',
+    },
+    {
+      value: '[{"name": "test", "channel": 1}]',
+    },
+    {
+      value: '[{"name": "test", "prerelease": true}]',
+    },
+    {
+      value: '[{"name": "test", "prerelease": 1}]',
+    },
+    {
+      value: '[{"name": "test", "range": true}]',
+    },
+  ])(
+    'throws an error if the input parameter is set to an invalid value %j',
+    ({ value }: { value: string }): void => {
+      expect.assertions(1);
+
+      getInputSpy.mockReturnValue(value);
+
+      // eslint-disable-next-line jest/require-to-throw-message
```

Comment:
This doesn't seem right. You can do `try-catch` and `expect(x).toBeInstanceOf(Error)`.